app/templates/base.html

```diff
@@ -1,45 +1,49 @@
 <!doctype html>
 <html lang="en">
-  {% block head %}
-    <!-- <script type="module"> is always `defer` -->
-    <script type="module">
-      import "http://localhost:5173/@vite/client";
-      window.process = { env: { NODE_ENV: "development" } };
-    </script>
-    <script type="module" src="http://localhost:5173/js/index.js"></script>
+  <head>
```

Comment:
change here was just to put the block within a `<head>` element. If you look at the raw html in the dev tools it seems like the doc didnt have a `<head>` and `<body>`